presto-orc/src/test/java/com/facebook/presto/orc/TestDecryption.java

```diff
@@ -311,6 +312,39 @@ public void testSingleEncryptionGroupRowType()
                 outputColumns);
     }
 
+    @Test
+    public void testEncryptionGroupWithReversedOrderNodes()
```

Comment:
Can we also add the same test but with the direct order? I briefly looked through other tests cases, they all seems to have the same type for all columns.

Reply:
there are a couple tests that have multiple types (e.g. https://github.com/prestodb/presto/blob/master/presto-orc/src/test/java/com/facebook/presto/orc/TestDecryption.java#L315), but it's good to have a simple explicit test, so added that case.